Resources/Prototypes/SS220/Entities/Objects/Devices/centcomm_newpda.yml

```
@@ -0,0 +1,31 @@
+# © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+- type: entity
+  parent: CentcomPDA
+  id: CentcomFieldOfficerPDA
+  name: ПДА Дипломата ЦК
+  description: ПДА Дипломата Центрального Командования. Его изысканный стиль позволяет более утонченно проводить переговоры. Особым аргументом является ручка, которая находится внутри него.
```

```suggestion
  description: КПК Дипломата Центрального Командования. Его изысканный стиль позволяет более утонченно проводить переговоры. Особым аргументом является ручка, которая находится внутри.
```
и так понятно, что ручка внутри него, а не в кармане, так что последнее местоимение не нужно.